examples/db.json

```
@@ -1,52 +1,36 @@
 {
-	"TV1L0-VGC-1": {
-		"_id": "TV1L0-VGC-1",
-		"active": true,
-		"args": [],
-		"beamline": "DEMO_BEAMLINE",
-		"creation": "Tue Feb 27 10:41:25 2018",
-		"device_class": "ophyd.sim.SynAxis",
-		"kwargs": {
-			"name": "{{name}}"
-		},
-		"last_edit": "Thu Apr 12 14:40:08 2018",
-		"macros": null,
-		"name": "tv1l0_vgc_1",
-		"parent": null,
-		"prefix": "TV1L0-VGC-1",
-		"screen": null,
-		"stand": "ST1",
-		"system": "Device",
-		"type": "Vacuum",
-		"z": 1035.5,
-		"invalid": true,
-		"functional_group": "Vacuum",
-		"location_group": "S 1"
-	},
-
-	"TV1L0-VGC-2": {
-		"_id": "TV1L0-VGC-2",
-		"active": true,
-		"args": [],
-		"beamline": "DEMO_BEAMLINE",
-		"creation": "Tue Feb 27 10:41:25 2018",
-		"device_class": "ophyd.sim.SynAxis",
-		"kwargs": {
-			"name": "{{name}}"
-		},
-		"last_edit": "Thu Apr 12 14:40:08 2018",
-		"macros": null,
-		"name": "tv1l0_vgc_2",
-		"parent": null,
-		"prefix": "TV1L0-VGC-2",
-		"screen": null,
-		"stand": "ST1",
-		"system": "Device",
-		"type": "Vacuum",
-		"z": 1035.6,
-		"invalid": true,
-		"functional_group": "Vacuum",
-		"location_group": "S 2"
-	}
-
+    "tv1l0_vgc_1": {
+        "_id": "tv1l0_vgc_1",
+        "active": true,
+        "args": [
+            "{{prefix}}"
+        ],
+        "creation": "Thu Jul 23 17:05:48 2020",
+        "device_class": "ophyd.sim.SynAxis",
+        "documentation": null,
+        "kwargs": {
+            "name": "{{name}}"
+        },
+        "last_edit": "Thu Jul 23 17:05:48 2020",
+        "name": "tv1l0_vgc_1",
+        "prefix": "TV1L0-VGC-1",
+        "type": "OphydItem"
+    },
+    "tv1l0_vgc_2": {
+        "_id": "tv1l0_vgc_2",
+        "active": true,
+        "args": [
+            "{{prefix}}"
+        ],
+        "creation": "Thu Jul 23 17:06:44 2020",
+        "device_class": "ophyd.im.SynAxis",
```

Is `validate()` broken? I don't think this should work...
```suggestion
        "device_class": "ophyd.sim.SynAxis",
```

---

Thanks for catching that @klauer - I tried it again and it seems to work with `ophyd.im.SynAxis` - maybe I am doing something wrong? Please let me know

```
>>> import happi
>>> client = happi.Client(path='db.json')
>>> client.validate()
[]
>>> client.search(type='OphydItem')
[SearchResult(client=<happi.client.Client object at 0x7f9fbddac850>, metadata={'name': 'tv1l0_vgc_1', 'device_class': 'ophyd.sim.SynAxis', 'args': ['{{prefix}}'], 'kwargs': {'name': '{{name}}'}, 'active': True, 'documentation': None, 'prefix': 'TV1L0-VGC-1', '_id': 'tv1l0_vgc_1', 'creation': 'Thu Jul 23 17:05:48 2020', 'last_edit': 'Thu Jul 23 17:05:48 2020', 'type': 'OphydItem'}), SearchResult(client=<happi.client.Client object at 0x7f9fbddac850>, metadata={'name': 'tv1l0_vgc_2', 'device_class': 'ophyd.im.SynAxis', 'args': ['{{prefix}}'], 'kwargs': {'name': '{{name}}'}, 'active': True, 'documentation': None, 'prefix': 'TV1L0-VGC-2', '_id': 'tv1l0_vgc_2', 'creation': 'Thu Jul 23 17:06:44 2020', 'last_edit': 'Thu Jul 23 17:06:44 2020', 'type': 'OphydItem'})]```